cli/update.go

```diff
@@ -52,7 +51,7 @@ func (uc *UpdateCommand) addFlags() {
 	flagSet.StringSliceVarP(&uc.env, "env", "e", nil, "Set environment variables for container")
 	flagSet.StringSliceVarP(&uc.labels, "label", "l", nil, "Set label for container")
 	flagSet.StringVar(&uc.restartPolicy, "restart", "", "Restart policy to apply when container exits")
-	flagSet.StringVar(&uc.quota, "quota", "", "Update disk quota for container")
+	flagSet.StringSliceVar(&uc.diskQuota, "disk-quota", nil, "Update disk quota for container")
```

Comment:
I think we should add more details here in the command line usage. For exapmle, how to unset the original diskquota @rudyfly